horovod/spark/lightning/remote.py

```diff
@@ -123,15 +123,15 @@ def train(serialized_model):
                 train_logger = TensorBoardLogger(logs_path)
                 print(f"Setup logger: Using TensorBoardLogger: {train_logger}")
 
-            elif isinstance(logger, CometLogger) and logger._experiment_key is None:
-                # Resume logger experiment key if passed correctly from CPU.
+            elif isinstance(logger, CometLogger) and logger_experiment_key is None:
```

Comment:
Did you mean "is not None"? this looks like it's creating a new CometLogger with an empty experiment.

Reply:
Good catch, yes, should be is not none.